AcloConcurrency/AcloConcurrency/Collection/ViewModel/CollectionViewModel.swift

```diff
@@ -0,0 +1,48 @@
+import Combine
+
+public class CollectionViewModel: ObservableObject {
+
+    @Published private(set) var mockArray = placesMock
+    @Published private(set) var places: [CollectionModel] = []
```

Comment:
```
@Published private(set) var places: [CollectionModel] = .mock
```

Reply:
почему через точку? Типо инитить сразу мок, а потом перезаписывать?

Reply:
Because when you will start get real values, you just replace .mock on your values and don't forget to delete mockArray or something. And code will be clearly